lib/msal-electron-proof-of-concept/samples/msal-electron-sample-app/src/Main.ts

```diff
@@ -0,0 +1,55 @@
+import { app, BrowserWindow } from 'electron';
+import { PublicClientApplication } from 'msal-electron-poc';
+import * as path from 'path';
+
+export default class Main {
```

Comment:
Is this an Electron convention to name the class "Main"? If not it is too in the nose :-)

Reply:
In this specific case it is. Electron apps have two main processes, the "Main" process (the sort of back end with access to Node API) and the "Renderer" process (fontend/ HTML and UI stuff.